data/en/lgbtq.yml

```diff
@@ -162,6 +173,7 @@
     - pseudo hermaphroditic
 - type: simple
   source: http://www.isna.org/node/979
+  note: These terms are stigmatizing to patients and their families. Intersex status is more complicated than the mere presence or absence of certain gonadal tissues.
```

Comment:
```suggestion
  note: These terms are stigmatizing to patients and their families because intersex status is more complicated than the mere presence or absence of certain gonadal tissues
```